Header header
float32 x
float32 y
float32 theta

float32 x_velocity
float32 y_velocity
float32 theta_velocity

bool all_motors_in_position
bool lookup_table_move_complete